app/views/proprietor/accounts/_shared_search_toggle.html.erb

```diff
@@ -0,0 +1,71 @@
+
+<!-- Originally from https://github.com/samvera/hyrax/blob/main/app/views/hyrax/admin/features/index.html.erb -->
+<% feature_set = all_features  %>
+<% grouped_features = feature_for_display_in_proprietor_account_ui %>
+
+<div class="flip row">
+  <div class="col-md-12">
+    <div class="panel">
+      <div class="panel-body">
+        <div class="table-responsive">
+          <table class="table table-striped">
+            <thead>
+              <tr>
+                <th></th>
+                <th class="name"><%= t('.feature') %></th>
+                <th class="description"><%= t('.description') %></th>
+                <th class="action"><%= t('.action') %></th>
+              </tr>
+            </thead>
+            <tbody>
+            <%# grouped_features&.each do |group, features| -%>
+            <% grouped_features&.each do |feature| -%>
+
+              <% if feature_set.grouped? -%>
+                <tr class="group">
+                  <td></td>
+                  <td class="name" colspan="<%= 2 + feature_set.strategies.size -%>">
+                    <h2>
+                      <%# t(group ? group.name : :default, scope: [:flipflop, :groups], default: group ? group.title : nil) -%>
+                    </h2>
+                  </td>
+                </tr>
+              <% end -%>
+              <%# features&.each do |feature| %>
+              <tr data-feature="<%= feature.name.dasherize.parameterize %>">
```

Comment:
This could also go into a partial to make the parent one easier to read

Reply:
let's let stay as it is in hyrax, so it is easy to follow changes.